nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/PutDatabaseRecord.java

```diff
@@ -265,6 +265,17 @@
             .expressionLanguageSupported(ExpressionLanguageScope.VARIABLE_REGISTRY)
             .build();
 
+    static final PropertyDescriptor BATCH_SIZE = new PropertyDescriptor.Builder()
```

Comment:
We should be consistent here with "batch size" and "bulk size" in the naming of variables, documentation, etc. Maybe "Maximum Batch Size"?

Reply:
Oh. I see it now. The display label is "Bulk Size". I'll fix it to be "Maximum Batch Size". Thanks